seed/views/main.py

```diff
@@ -284,17 +286,18 @@ def export_buildings_progress(request):
     percent_done = progress_data['progress']
     total_buildings = progress_data['total_buildings']
 
-    return {
+    return JsonResponse({
```

Comment:
Same thing here, I think `request.body` will fail to work here, right?  Is it working and I'm just bonkers??